src/cljc/rems/catalogue_util.cljc

```diff
@@ -1,9 +1,9 @@
 (ns rems.catalogue-util
   (:require [clojure.string :as str]))
 
-(defn urn-catalogue-item? [{:keys [resid]}]
-  (and resid (str/starts-with? resid "urn:")))
+(defn urn? [resid]
```

Comment:
nit: This could be private now.

Reply:
yeah, but it's a potentially useful util elsewhere too

Reply:
Ok, fine by me to merge.